src/components/volume-control/volume-control.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { SpeakerIcon } from '../../assets/icons/speaker-icon/speaker-icon';
+import './volume-control.scss';
+
+interface VolumeControlProps {
+  volume: string;
```

Comment:
i feel like this should just be a `number` then internally we can cast it back to a string
and maybe it can be named better to `volumePercentage` (?) to hint it's a number from 0-100

then, you can also clamp this value (using your `clamp` function) which should maybe be moved to `helpers/func.ts`

Reply:
That's a good idea. I'll get that fixed.